pkg/kv/kvserver/replica_sideload_test.go

```diff
@@ -198,7 +199,7 @@ func TestRaftSSTableSideloading(t *testing.T) {
 	var idx int
 	for idx = 0; idx < len(ents); idx++ {
 		// Get the SST back from the raft log.
-		if !logstore.SniffSideloadedRaftCommand(ents[idx].Data) {
+		if v, _, _ := kvserverbase.DecodeRaftCommand(ents[idx].Data); v != kvserverbase.RaftVersionSideloaded {
```

Comment:
Should this also check the type? Could reuse the same helper.

Reply:
Hmm, narrowly came down on the side of no for this, as of the latest code a helper would allow us to write this

```go
	ok, err := isRaftVersionSideloaded(ent)
	if err != nil {
		return nil, err
	}
	if !ok {
		return nil, nil
	}
```

instead of this:

```go
	typ, err := raftlog.EncodingVersion(ent)
	if err != nil {
		return nil, err
	}
	if typ != kvserverbase.RaftVersionSideloaded {
		return nil, nil
	}
```

which doesn't seem worth it. If we didn't have to handle the error it would have been, but I don't think we should swallow it.